types/module/module_test.go

```diff
@@ -79,8 +79,6 @@ func TestGenesisOnlyAppModule(t *testing.T) {
 
 	// no-op
 	goam.RegisterInvariants(mockInvariantRegistry)
-	goam.BeginBlock(sdk.Context{}, abci.RequestBeginBlock{})
```

Comment:
wanna make sure that these checks are not require now that we removed all the unused `BeginBlock` and `EndBlock` 

cc @alexanderbez @marbar3778